src/components/Modal/src/Modal.vue

```diff
@@ -1,16 +1,44 @@
 <template>
-	<div :class="$s.Modal">
+	<div
+		:class="$s.Modal"
+		:style="modalStyles"
+	>
+		<template v-if="$slots.dismiss">
```

Comment:
I think we could simplify this by always having the the touch capture wrap the default slot. Only have one of them (don't need separate ones for mobile / desktop) and then simply hide invisible container on desktop.

Reply:
I'm not sure why but the PRs on maker don't  auto-hide code that's outdated...just adds an Outdated tag. But all the code around dismiss slots have been removed - the actions are just on the modal itself...there are no touch actions on desktop so it's only on mobile.

Reply:
Oh weird! I might have dug into a cached commit and didn't see the updates.